### ISSUE TITLE
Please add bookmarks to an existing folder, no need to create a new folder.

### ISSUE THREAD

**Issue by billng** [eggplanetio/tidytab#110]
As above

**Comment by briangonzalez** [comment]
Can you explain further? Do you want this to be configurable?

**Comment by billng** [comment]
Yes, it would be great if this feature can be configurable. Every time by clicking TidyTab button, the bookmarks are saved into a new created folder. But I prefer to save bookmarks into the same existing folder, so that I can easily find my unread tabs (the saved bookmarks) later from Chrome in the mobile phone instead of only in PC.
